cmd/mev-boost/main.go

```diff
@@ -21,16 +24,23 @@ var (
 	// cli flags
 	port         = flag.Int("port", defaultPort, "port for mev-boost to listen on")
 	executionURL = flag.String("executionUrl", defaultExecutionURL, "url to execution client")
-	relayURL     = flag.String("relayUrl", defaultRelayURL, "url to relay")
+	relayURL     = flag.String("relayUrl", defaultRelayURL, "url to relay (single entry or comma-separated list)")
```

Comment:
rename everywhere to `relayUrls`, including the CLI param

Reply:
will do. wasn't sure because that's a breaking change